### PR TITLE
Allow passing a function to Appsignal.send_error/6

### LINKED ISSUE #85: Allow sending sample data via Appsignal.send_error/5

### PR DESCRIPTION
**Pull request by jeffkreeftmeijer**
Closes #85.

### CODE REVIEW

File: lib/appsignal.ex [hunk 1]
```diff
@@ -115,10 +115,11 @@ defmodule Appsignal do
   When there is no current transaction, this call starts one.
 
   """
-  def send_error(reason, message \\ "", stack \\ nil, metadata \\ %{}, conn \\ nil) do
+  def send_error(reason, message \\ "", stack \\ nil, metadata \\ %{}, conn \\ nil, fun \\ fn(t) -> t end) do
```
Comment (arjan) [inline lib/appsignal.ex]: It would be nice to improve the documentation of this function. But you can also open a separate issue for this.

Comment (tombruijn) [inline lib/appsignal.ex]: Docs yes!

We need to make a page like this one for Elixir: http://docs.appsignal.com/ruby/instrumentation/exception-handling.html

**Comment by jeffkreeftmeijer** [comment]
There's [a documentation project lined](https://github.com/appsignal/appsignal-docs/issues/69) up in the 1.0 milestone. I've added some usage examples in 8a9eaf8.

**Comment by jeffkreeftmeijer** [comment]
Added usage examples to the docs, this one's ready for re-review.

**Pull request merged**